apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/TestDatabaseFixture.cs

```diff
@@ -1,68 +1,78 @@
-﻿// Copyright 2017 Google Inc. All Rights Reserved.
-// 
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-// 
-//     http://www.apache.org/licenses/LICENSE-2.0
-// 
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+﻿#region
 
 using System;
 using System.Threading;
 using System.Threading.Tasks;
 using Google.Cloud.Spanner.Admin.Database.V1;
 
+#endregion
+
 namespace Google.Cloud.Spanner.Data.IntegrationTests
 {
-    public class TestDatabaseFixture : IDisposable {
-
-        private readonly string _databaseName = "t_" + Guid.NewGuid().ToString("N").Substring(0, 28);
-        private Task _creationTask;
+    public class TestDatabaseFixture : IDisposable
+    {
+        // scratch can be used to run tests on a precreated db.
+        // all tests are designed to be re-run on an exiting db (previously written state will
+        // not cause failures).
+        // if you use a scratch database, be sure to comment out the database
+        // creation and disposal methods.
+        private readonly string _databaseName = //"scratch";  
+            "t_" + Guid.NewGuid().ToString("N").Substring(0, 28);
         private readonly string _testTable = "TestTable";
+        private Task _creationTask;
         private DatabaseAdminClient _databaseAdminClient;
 
-        public void Dispose() {
-            if (_databaseAdminClient != null) {
-                _databaseAdminClient.DropDatabase(new DatabaseName(TestProjectName, TestInstanceName, DatabaseName));
-            }
-        }
-
         public string TestInstanceName => "myspanner";
 
         public string TestProjectName => "spanneref";
 
-        public string ConnectionString => "Data Source=" + TestProjectName+ "/" + TestInstanceName+ "/"
-                                          + DatabaseName;
+        public string ConnectionString => "Data Source=" + TestProjectName + "/" + TestInstanceName
+                                          + "/" + DatabaseName;
```

Comment:
I'm syncing with Jim and I'm going to check in the settings file with the project.
A lot of the code feedback (whitespace, explicit visibility, preference of body expressions) can all be automatically done by resharper.  The trick is to get the settings file to correspond to what we want -- or else it will constantly be changing code to a different style.
Mind if I follow up with whitespace and other fixes with another PR?